src/main/java/dal/fatih/todoly/Main.java

```diff
@@ -1,18 +1,22 @@
 package dal.fatih.todoly;
 
 import java.text.DateFormat;
+import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.util.*;
 
 
-
 public class Main {
 
-   /* public static String dateFormatter(Date date) {
-        DateFormat dueDateFormat = new SimpleDateFormat("dd.MM.yyyy");
-        String resultDate = dueDateFormat.format(date);
-        return resultDate;
-    }*/
+    /* public static String dateFormatter(Date date) {
+         DateFormat dueDateFormat = new SimpleDateFormat("dd.MM.yyyy");
+         String resultDate = dueDateFormat.format(date);
+         return resultDate;
+     }*/
+    public static Date dateParser(String date) throws ParseException {

```

Comment:
The name is not correct. Method names MUST be a verb. `dateParser` is a noun, make it something like `parseDate`